### DISCUSSION TITLE
Postgres Security with Prisma, How?

### DISCUSSION THREAD

**Discussion by darrylmorley** [prisma/prisma#9507] [Q&A]
I was trying to add a little security to my PostgreSQL database today, but it seem nothing works with `prisma db pull` unless there is full superuser rights?

Can anybody offer any insight as to what postgres permissions are required for prisma to work?

Thanks

**Comment by janpio** [comment]
What errors are you getting? What right exactly did you give your user?

**Reply by darrylmorley** [reply]
I gave the Postgres user CONNECT, SELECT & LOGIN. I can't remember the specific error, but it simply wouldn't connect to the database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
